static/app/views/replays/types.tsx

```diff
@@ -29,9 +30,9 @@ export type ReplayRecord = {
   };
   dist: null | string;
   /**
-   * Difference of `updated-at` and `created-at` in seconds.
+   * Difference of `finishedAt` and `startedAt` in seconds.
    */
-  duration: number; // Seconds
+  duration: Duration;
```

Comment:
Change the type so that typescript tells me all the places that depend on it, and remove the ambiguity of what unit is it (sec or ms)

Reply:
but also.. moment is here now.

Reply:
Yeah unfortunately lots of moment used throughout the app